src/ncp/ncp_base_mtd.cpp

```diff
@@ -182,6 +182,9 @@ otError NcpBase::EncodeNeighborInfo(const otNeighborInfo &aNeighborInfo)
     SuccessOrExit(error = mEncoder.WriteUint32(aNeighborInfo.mLinkFrameCounter));
     SuccessOrExit(error = mEncoder.WriteUint32(aNeighborInfo.mMleFrameCounter));
     SuccessOrExit(error = mEncoder.WriteInt8(aNeighborInfo.mLastRssi));
+#if OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2
+    SuccessOrExit(error = mEncoder.WriteUint32(aNeighborInfo.mLinkAckFrameCounter));
```

Comment:
If we update this, I think we should also update the `spinel.h` and the documentation of this property